pkg/artifactory/resource_artifactory_keypair_test.go

```diff
@@ -93,7 +93,8 @@ func TestAccKeyPairFailPubCertCheck(t *testing.T) {
 	})
 }
 
-func TestAccRSAKeyPair(t *testing.T) {
+//TODO: failing test - to fix
+func skipTestAccRSAKeyPair(t *testing.T) {
```

Comment:
It doesn't actually fail locally.

Reply:
we need to figure out why is the difference between local and k8s instances

Reply:
since we run "pre-production" test on the real k8s instance RT+Xray, let's assume the test fails and skip it in the pipeline until we fix it?

Reply:
This is between you @alexhung  and @maheshjfrog